upup/models/cloudup/resources/addons/node-termination-handler-queue-processor.aws/k8s-1.11.yaml.template

```diff
@@ -0,0 +1,277 @@
+
+# Sourced from https://github.com/aws/aws-node-termination-handler/releases/download/v1.12.1/all-resources-queue-processor.yaml
+---
+# Source: aws-node-termination-handler/templates/psp.yaml
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
```

Comment:
Technically the PSP stuff isn't required. kube-system has a default allow-anything policy if the admission controller is enabled. But fine to leave it in if it makes copy/pasting from source easier.

Reply:
I'm leaning towards leaving it in. IMO it'd be simpler to keep it exactly in line with the default config released by NTH

Reply:
I ended up removing it and merging the two templates, with an if/else to switch between the IMDS daemonset and the Queue Processor deployment, the rest of the templates were the same